### ISSUE TITLE
Scheme time not showing correctly 

### ISSUE THREAD

**Issue by mattfox27** [nielsfaber/scheduler-card#704]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

Latest 

### Bug description

When I setup a scheme for instance this one turns lights on at 9am and off at 9am it says midnight when I actually look at the schedule card 
![20230506_092324](https://user-images.githubusercontent.com/6801877/236635764-272c3607-ae24-4e11-ae69-0ba005be6ac7.jpg)

### Steps to reproduce

Set scheme up for 9am-9pm

**Comment by nielsfaber** [comment]
A schedule spans across one day, so from 12:00 AM to 11:59 PM.
At midnight the schedule jumps from the last timeslot to the first.
If you have actions assigned to the first and last timeslot, this means at midnight an action is planned.
It could be that the action has no effect (if the light is already off it stays off), but there is no intelligence built in to detect whether this can be skipped.

In other words, this is not a bug but just a side effect of the schedules repeating the next day.
